plate-resort-multiple/prefect_flows/deploy_flows.py

```
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+"""
+Helper script to deploy all Plate Resort flows to a work pool.
+Run this script after creating the work pool to deploy all flows.
+"""
+import sys
+from device import (
+    connect_flow,
+    disconnect_flow,
+    status_flow,
+    health_flow,
+    activate_hotel_flow,
+    go_home_flow,
+    move_to_angle_flow,
+    set_speed_flow,
+    emergency_stop_flow,
+    get_hotels_flow,
+    get_position_flow,
+)
+
+
+def deploy_all(work_pool_name: str = "plate-resort-pool"):
+    """Deploy all flows to the specified work pool"""
+    flows = [
+        (connect_flow, "plate-resort-connect"),
+        (disconnect_flow, "plate-resort-disconnect"),
+        (status_flow, "plate-resort-status"),
+        (health_flow, "plate-resort-health"),
+        (activate_hotel_flow, "plate-resort-activate-hotel"),
+        (go_home_flow, "plate-resort-go-home"),
+        (move_to_angle_flow, "plate-resort-move-to-angle"),
+        (set_speed_flow, "plate-resort-set-speed"),
+        (emergency_stop_flow, "plate-resort-emergency-stop"),
+        (get_hotels_flow, "plate-resort-get-hotels"),
+        (get_position_flow, "plate-resort-get-position"),
+    ]
+
+    print(f"Deploying {len(flows)} flows to work pool: {work_pool_name}")
+    print("-" * 60)
+
+    for flow, deployment_name in flows:
+        try:
```

I don't like the try except here. Let error bubble naturally so we can debug.